python/cutlass/utils/lazy_import.py

```diff
@@ -0,0 +1,11 @@
+import importlib
+from typing import Any
+
+def lazy_import(mod_name: str) -> Any:
+    class Lazy:
+        def __getattr__(self, name:str) -> Any:
+            module = importlib.import_module(mod_name)
```

Comment:
You can just join name with module to get the attribute, right? No need to call getattr